src/core/server/index.ts

```diff
@@ -115,7 +115,7 @@ export interface InternalCoreSetup {
  * @public
  */
 export interface InternalCoreStart {
-  http: HttpServiceStart;
+  // http: HttpServiceStart;
```

Comment:
I'm not sure why this isn't needed anymore, but just remove it entirely rather than commenting out.

Reply:
I will remove it. we cannot export `isListening` anymore because `http server` is the last server to run currently. the use case for `secondary server` should be separated from this method https://github.com/restrry/kibana/blob/master/src/core/server/http/http_service.ts#L106-L107